persistence-test/src/main/java/io/stargate/it/PersistenceTest.java

```diff
@@ -1964,6 +1964,31 @@ public void testRowDecorator() throws ExecutionException, InterruptedException {
     assertGt(last, first, dec1, dec2);
   }
 
+  @Test
+  public void testRowDecoratorComparableBytes() throws ExecutionException, InterruptedException {
+    setupCustomPagingData();
+
+    // Obtain partition keys in "ring" order
+    AbstractBound<?> selectAll =
+        dataStore
+            .queryBuilder()
+            .select()
+            .column("pk")
+            .column("val")
+            .from(keyspace, table)
+            .build()
+            .bind();
+
+    ResultSet rs1 = dataStore.execute(selectAll).get();
+    RowDecorator dec1 = rs1.makeRowDecorator();
+    ByteBuffer src = dec1.getComparableBytes(rs1.one());
+    if (backend.isDse()) {
+      assertThat(src.array().length).isGreaterThan(0);
+    } else {
+      assertThat(src.array().length).isEqualTo(0);
+    }
+  }
```

Comment:
this is not enough, we need the same test as in the ` testRowDecorator`, just confirming that bytes are created is not enough, we need to confirm that comapring is done correctly.. should be one-to-one with the `testRowDecorator` and then test comparing using a unsigned bytes comparator .. And sure only active for DSE now..

Reply:
I added a better test for this - but in order to do so I had to have `ByteComparable` in scope and so had to add a DSE dependency to the persistence-test module...hope that's ok.

Maybe it can be reconfigured to _only_ import that package